src/Umbraco.Tests.Integration/Umbraco.Core/Packaging/CreatedPackagesRepositoryTests.cs

```diff
@@ -259,15 +277,24 @@ public void Export_Zip()
             using (FileStream packageZipStream = File.OpenRead(packageXmlPath))
             using (ZipArchive zipArchive = PackageMigrationResource.GetPackageDataManifest(packageZipStream, out XDocument packageXml))
             {
-                Assert.AreEqual("umbPackage", packageXml.Root.Name.ToString());
-                Assert.IsNotNull(zipArchive.GetEntry("media/media/test-file.txt"));
-
-                Assert.AreEqual(
-                    $"<MediaItems><MediaSet><testImage id=\"{m1.Id}\" key=\"{m1.Key}\" parentID=\"-1\" level=\"1\" creatorID=\"-1\" sortOrder=\"0\" createDate=\"{m1.CreateDate.ToString("s")}\" updateDate=\"{m1.UpdateDate.ToString("s")}\" nodeName=\"Test File\" urlName=\"test-file\" path=\"{m1.Path}\" isDoc=\"\" nodeType=\"{mt.Id}\" nodeTypeAlias=\"testImage\" writerName=\"\" writerID=\"0\" udi=\"{m1.GetUdi()}\" mediaFilePath=\"/media/test-file.txt\"><umbracoFile><![CDATA[/media/test-file.txt]]></umbracoFile><umbracoBytes><![CDATA[100]]></umbracoBytes><umbracoExtension><![CDATA[png]]></umbracoExtension></testImage></MediaSet></MediaItems>",
-                    packageXml.Element("umbPackage").Element("MediaItems").ToString(SaveOptions.DisableFormatting));
-
-                // TODO: There's a whole lot more assertions to be done
-
+                string test = "test-file.txt";
+                Assert.Multiple(() =>
+                {
+                    Assert.AreEqual("umbPackage", packageXml.Root.Name.ToString());
+                    Assert.IsNotNull(zipArchive.GetEntry("media/media/test-file.txt"));
+                    Assert.IsNotNull(zipArchive.GetEntry("package.xml"));
+                    Assert.AreEqual(
+                        $"<MediaItems><MediaSet><testImage id=\"{m1.Id}\" key=\"{m1.Key}\" parentID=\"-1\" level=\"1\" creatorID=\"-1\" sortOrder=\"0\" createDate=\"{m1.CreateDate.ToString("s")}\" updateDate=\"{m1.UpdateDate.ToString("s")}\" nodeName=\"Test File\" urlName=\"test-file\" path=\"{m1.Path}\" isDoc=\"\" nodeType=\"{mt.Id}\" nodeTypeAlias=\"testImage\" writerName=\"\" writerID=\"0\" udi=\"{m1.GetUdi()}\" mediaFilePath=\"/media/test-file.txt\"><umbracoFile><![CDATA[/media/test-file.txt]]></umbracoFile><umbracoBytes><![CDATA[100]]></umbracoBytes><umbracoExtension><![CDATA[png]]></umbracoExtension></testImage></MediaSet></MediaItems>",
+                        packageXml.Element("umbPackage").Element("MediaItems").ToString(SaveOptions.DisableFormatting));
+                    Assert.AreEqual(2, zipArchive.Entries.Count());
+                    Assert.AreEqual(ZipArchiveMode.Read, zipArchive.Mode);
+                    Assert.IsNull(packageXml.DocumentType);
+                    Assert.IsNull(packageXml.NextNode);
+                    Assert.IsNull(packageXml.Parent);
+                    Assert.IsNull(packageXml.PreviousNode);
+                    Assert.AreEqual(test, zipArchive.Entries.Last().Name);
+                    Assert.AreEqual(7, zipArchive.Entries.Last().Length);
+                });
```

Comment:
Here you first fetch the `test-file.txt` entry with `zipArchive.GetEntry("media/media/test-file.txt");`, later you fetch the same entry again when you do `Assert.AreEqual(test, zipArchive.Entries.Last().Name);`.

Instead you could just do something like:

```C#
var mediaEntry = zipArchive.GetEntry("media/media/test-file.txt")
Assert.IsNotNull(mediaEntry);
Assert.AreEqual(test, mediaEntry.Name);
Assert.AreEqual(7, mediaEntry.Length);
```

When that's said I don't think we should assert the uncompressed size of the entry, since this is bound to change on different platforms in a text file, for instance Microsoft uses CRLF for line endings where linux just uses LF, this will result in different file sizes, as evident by the Linux tests failing 😄